src/cljs/rems/navbar.cljs

```diff
@@ -60,6 +60,10 @@
               text (get-in page [:translations language :title] (text :t/missing))]
           [nav-link url text])))))
 
+(defn logo [menu-navigation?]
+  [:div {:class (str (if menu-navigation? "navbar-brand logo-menu" "logo"))}
+   [:div.img]])
+
```

Comment:
We could make this much simpler by having two different components instead of this parameterized component. But what I do not like is that this `menu-navigation?` parameter does a very unsymmetrical choice between adding two or one completely unrelated class. Why are the classes not `logo-navbar` and `logo`? Why do we have these `navbar-brand` and `logo-menu` here? Is the terminology clear? Why is there also `:logo-name-navigation`? Can we not have one term for the component in the navbar and use the same for the config etc?

Reply:
Bootstrap docs again

Reply:
https://getbootstrap.com/docs/4.0/components/navbar/

Reply:
I can accept using `navbar-brand` here but I think we shouldn't need the other reference to it (from the `styles.clj`).

I also suggest using `:logo-name-navbar` so that we don't need to remember `navbar` vs. `navigation` for this. We should be able to do away with `logo-menu` entirely. Then we would always refer to `navbar` and `logo` for these things (except the one reference to Bootstrap styling).

Reply:
Also
- `logo-name` is the name of the logo
- `logo-name-sm` is the name of the logo in small breakpoint

I suggest `navbar-logo-name` because it is not a variant of the logo but a separate thing for the navbar.

Reply:
A lot of Bootstrap native classes are currently being referenced in our ```styles.clj```  file.

Reply:
As of now, I did delete the reference to ```navbar-brand``` from ```styles.clj```